docs/source/getstarted.rst

```diff
@@ -27,12 +27,13 @@ Please visit our `Cozmo SDK Forums <https://forums.anki.com/>`_ where you can:
 Starting Up the SDK
 -------------------
 
-1. Plug the mobile device containing the Cozmo app into your computer.
-2. Open the Cozmo app on the phone. Make sure Cozmo is on and connected to the app via WiFi.
-3. Tap on the gear icon at the top right corner to open the Settings menu.
-4. Swipe left to show the Cozmo SDK option and tap the Enable SDK button.
-5. On the computer, open a Terminal (macOS/Linux) / Command Prompt (Windows) window. Type ``cd cozmo_sdk_examples`` where *cozmo_sdk_examples* is the name of the directory you extracted the SDK examples into and press Enter. The SDK example scripts can be downloaded from the :doc:`Downloads page </downloads>`.
-6. Now you can run any of the example programs.
+1.	Plug the mobile device containing the Cozmo app into your computer.
```

Comment:
Looks like the whitespace in this list between the numbers (for instance, "1.") and the text ("Plug the mobile device...") changes from what it was before. Let's keep the whitespace as is in case it results in changes on the page.